services/collaboration/pkg/config/http.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+import (
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+)
+
+// HTTP defines the available http configuration.
+type HTTP struct {
+	Addr      string                `yaml:"addr" env:"COLLABORATION_HTTP_ADDR" desc:"The external address of the HTTP service. Either IP address or host (127.0.0.1:9301 or wopi.private.prv). The configured "Scheme" will be used to build public URLs along with this address."`
+	BindAddr  string                `yaml:"bindaddr" env:"COLLABORATION_HTTP_BINDADDR" desc:"The bind address of the HTTP service."`
+	Namespace string                `yaml:"-"`
+	Scheme    string                `yaml:"scheme" env:"COLLABORATION_HTTP_SCHEME" desc:"Either http or https"`
```

Comment:
```suggestion
	Scheme    string                `yaml:"scheme" env:"COLLABORATION_HTTP_SCHEME" desc:"The scheme to use. Either 'http' or 'https'."`
```
Q: this defines the scheme for `COLLABORATION_HTTP_BINDADDR` - or? In case yes, we must add a note about this in that envvar.